src/helix-ui/elements/HXTabsetElement.js

```diff
@@ -1,99 +1,193 @@
 import { HXElement } from './HXElement';
-import shadowStyles from './_hx-tabset.less';
-
-const tagName = 'hx-tabset';
-const template = document.createElement('template');
-
-template.innerHTML = `
-    <style>${shadowStyles}</style>
-    <div id ="head">
-      <slot name="tabs"></slot>
-    </div>
-    <div id ="body">
-      <slot></slot>
-    </div>
-`;
+import { KEYS } from '../util';
 
 export class HXTabsetElement extends HXElement {
     static get is () {
-        return tagName;
+        return 'hx-tabset';
     }
 
     static get observedAttributes () {
-        return [ 'selected' ];
+        return [ 'current-tab', 'tab-side' ];
```

Comment:
Remove `tab-side` (it was copied from somewhere, but we only care about top not side)